theme/src/components/nav-drawer.js

```diff
@@ -10,6 +11,37 @@ import Details from './details'
 import Drawer from './drawer'
 import NavItems from './nav-items'
 
+function useNavDrawerState(breakpoint) {
```

Comment:
[Minor] For named exports, I usually put the `export` keyword before the function declaration instead of at the end of the file but I don’t have a strong opinion about it. Eventually, we might want to have guidelines around how we prefer to export things (e.g. named exports vs. default exports, etc).